### ISSUE TITLE
Solana SDK Connections

### ISSUE THREAD

**Issue by Elaixzar** [Glitter-Finance/SDK#54]
Please redo connects so that all 3 (Mainnet, testnet and devnet) are available and we can override the default RPCs of each one in the config file.

**Comment by saadjhk** [comment]
@Elaixzar this feels like a breaking change, any chance we can split this into multiple smaller releases?

**Comment by AndrewC-Glitter** [comment]
@saadjhk, yes.  I've already started it for single RPC/chain. (Needed for server)
